lib/src/main/java/com/orange/ods/compose/component/banner/OdsBanner.kt

```diff
@@ -185,6 +209,7 @@ private val previewParameterValues: List<OdsBannerPreviewParameter>
 
         return listOf(
             OdsBannerPreviewParameter(longMessage, firstButtonText, secondButtonText, imageRes),
+            OdsBannerPreviewParameter(shortMessage),
```

Comment:
The text (`"Two lines text string with two actions."`) is not adapted to this case because there is no action here.